src/repl.jl

```diff
@@ -235,3 +237,20 @@ function fixjunodisplays()
   end
   pushdisplay(JunoDisplay())
 end
+
+# completions
+
+struct JunoREPLCompletionProvider <: REPL.CompletionProvider
+  mod::Module
+end
+
+function LineEdit.complete_line(c::JunoREPLCompletionProvider, s)
+  partial = REPL.beforecursor(s.input_buffer)
+  full = LineEdit.input_string(s)
+
+  # module-aware repl backend completions
+  comps, range, should_complete = REPLCompletions.completions(full, lastindex(partial), c.mod)
+  ret = map(REPLCompletions.completion_text, comps) |> unique!
```

Comment:
I may want to show `ModuleCompletion`s with its module identical to `JunoREPLCompletionProvider.mod` up: this won't be so computational heavy but what do you think about this idea ?

Reply:
Sorry, not sure what you mean.

Reply:
NVM... I had a kinda same idea within debugger local completions, but now I find there is no need for this here at all.